src/app/cart/templates/cart.tpl.html

```diff
@@ -18,6 +18,14 @@ <h3 class="panel-title">
                     </h3>
                 </div>
                 <div class="panel-body">
+                    <div class="well text-center">
+                        <div ng-if="base.currentOrder.Total >= 250">
+                            Your order $250 or over will be shipped freight free
```

Comment:
This sentence is a little confusing to read. Can we just say your order will be shipped free? I guess if thats what they are requesting then this is fine